### DISCUSSION TITLE
[Request] Allow third party pack launchers

### DISCUSSION THREAD

**Discussion by HalJordan** [NovaMachina-Mods/ExNihiloSequentia#362] [Ideas]
### Is there an existing suggestion for this?

- [X] I have searched the existing suggestions

### Feature Description

What reason could you possibly have for blocking this? Certainly it's not the default and you forgot?

### Provided Benefit

Less headache when trying to play popular packs

**Comment by AlfredoPorfaverta** [comment]
Please allow this. Im trying to make a modded server with this mod but I can't since it doesn't allow third party things

**Comment by dsnidr** [comment]
This is still an issue in 2023. Any plans to allow third party launchers? It would really make installing modpacks easier.

**Reply by HalJordan** [reply]
Or could we at least get a positive "no" response instead of being left in limbo a year later??

**Reply by NovaMachina** [reply]
Just went through and allowed third party distribution through CurseForge. Do note that everything is available through Modrinth as well.

**Comment by NovaMachina** [comment]
Apologies, I thought this had already been answered. Ex Nihilo: Sequentia is available through Modrinth and CurseForge. Is there still a need for 3rd party through CurseForge?

**Comment by NovaMachina** [comment]
Just went through and allowed third party distribution through CurseForge. Do note that everything is available through Modrinth as well.
